mifospay/src/main/java/org/mifos/mobilewallet/mifospay/auth/ui/LoginActivity.java

```diff
@@ -101,7 +101,7 @@ private void handleLoginInputChanged() {
 
     @OnClick(R.id.btn_login)
     public void onLoginClicked() {
-        Utils.hideSoftKeyboard(this);
+        Utils.INSTANCE.hideSoftKeyboard(this);
```

Comment:
Do we really need `INSTANCE` here and in other files, can't we call the method directly like: `Utils.hideSoftKeyboard(this);`?